About us

lovebaihezi opened the pull request:
web的图片还没有更新,现在用的还是老的

Commit:
fix: 🐛 not clear timer will cause memory leak

settimeout not clear will cause current fibernode still stay in memory,
with a hook call, this fiber node will cause memory leak

BREAKING CHANGE: 🧨 none

✅ Closes: #19

Commit:
feat: 🎸 about us

add about us,left web img not update

BREAKING CHANGE: 🧨 none

✅ Closes: none

Commit:
feat: 🎸 add pictures

update intro and images

BREAKING CHANGE: 🧨 no

✅ Closes: no

Commit:
Merge branch 'master' into about-us

kibuniverse commented on src/pages/about-us/index.tsx:
```diff
@@ -1,11 +1,79 @@
+  </>
+  )
+
+const AboutPicture: FC<{ picture_url: string }> = ({ picture_url }) => (
```
这组件分的也太细了

kibuniverse replied:
分的细本身没啥问题，但是没啥逻辑并且复用度不高的话，就直接写在一个里面吧

kibuniverse commented on src/pages/about-us/introduction.ts:
```diff
@@ -0,0 +1,42 @@
+const dir = 'src/common/pictures/'
```
这个文件放在全局的 constants.ts 里 作为静态配置

kibuniverse reviewed (commented):
add some comment

lwsys committed:
!2 submit
Merge pull request !2 from shisifan/home-fix

Commit:
refactor: 💡 change the config file to @/common/constant.ts

all the config should manage by the center dir

BREAKING CHANGE: 🧨 none

✅ Closes: none

Commit:
Merge remote-tracking branch 'update/master' into about-us
update the about-us with update/master (remote)
